recipe/meta.yaml

```
@@ -68,6 +68,7 @@ outputs:
         - entrypoints
         - platformdirs
         - pydantic
+        - stamina
         - rich
```

Alphabetical sorting:
```suggestion
        - rich
        - stamina
```